src/components/CategoricalColorPicker.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ReactElement, useMemo } from "react";
+import { Dataset } from "../colorizer";
+import { ColorPicker, Tooltip } from "antd";
+import { Color, ColorRepresentation } from "three";
+
+import { Color as AntColor } from "antd/es/color-picker/color";
+import styled from "styled-components";
```

Comment:
nit: it seems like you've grouped these by 3rd party and local, but this and `Dataset` are in the wrong place

Reply:
Thanks, good catch! I need to set up prettier to do this for me 😂